docs/events.rst

```diff
@@ -74,7 +78,9 @@ It's also possible to dump the raw events using the API.
        }
     ]
 
-The ``set`` parameter accepts an optional ``union`` or ``intersection`` argument to determine the behavior for filtering sets of tags (i.e. inclusive or exclusive). By default, Graphite uses a "lazy union" that will return any matching events for a given tag in a list of tags. This behavior is not intuitive and will therefore be deprecated in a future release.
+The ``set`` parameter accepts an optional ``union`` or ``intersection`` argument to determine the behavior for filtering sets of tags (i.e. inclusive or exclusive). By default, Graphite uses a "lazy union" that will return any matching events for a given tag in a list of tags. This behavior is not intuitive and will therefore be deprecated in a future release. When matching using multiple tags, separate the tags with a space.
+
+*Note*: Prior to 0.10.0, the value of ``tags`` will be a string, with multiple tags being separated by a space.
```

Comment:
Sorry - I had meant to remove that sentence. I'll remove it. (the "When matching...")

About the output, I only have it there because it changed - in 0.9, the output doesn't match the example. It seemed worth noting. If you feel that has no value, I will remove it as well.